README.rst

```diff
@@ -21,16 +21,18 @@
 Project info
 ------------
 
-The Scikit-HEP project (http://scikit-hep.org/) is a community-driven and community-oriented project
-with the aim of providing Particle Physics at large with a Python package containing core and common tools.
-The project started in Autumn 2016 and its packages are actively developed and maintained.
+The `Scikit-HEP project <http://scikit-hep.org/>`_ is a community-driven and community-oriented project
+with the aim of providing Particle Physics at large with an ecosystem for data analysis in Python
+embracing all major topics involved in a physicist's work.
+The project started in Autumn 2016 and is in full swing.
```

Comment:
I liked the old final sentence better. "full swing" doesn't really mean anything concrete, but "packages are actively developed and maintained" does.

Reply:
OK, will revert that bit. I agree with you, BTW.